package.json

```diff
@@ -7,11 +7,14 @@
     "build:sassdoc": "sassdoc --parse node_modules/govuk-frontend/govuk/ > data/sassdoc.json",
     "scripts": {
       "lint": "standard"
-    }
+    },
+    "test": "hyperlink -ri deploy/public/index.html | tap-spot"
```

Comment:
Would it be possible to document what the `-ri` option does? I can't find it in the library docs.